Concierge/Expose-Bank/Contextual_Theming/Base.lproj/Main.storyboard

```diff
@@ -0,0 +1,114 @@
+                            <constraint firstItem="nV2-0l-3tf" firstAttribute="bottom" secondItem="8bC-Xf-vdC" secondAttribute="bottom" id="fAK-NU-5IY"/>
+                        </constraints>
+                    </view>
+                    <navigationItem key="navigationItem" id="bu4-4M-8ih"/>
+                    <connections>
+                        <outlet property="firstView" destination="miR-cu-Ipc" id="TZ4-JB-OUW"/>
+                        <outlet property="secondView" destination="XZ4-zZ-4NG" id="j0W-wl-cYo"/>
+                        <outlet property="thirdView" destination="y5u-2m-Con" id="B7S-TD-7rd"/>
+                    </connections>
+                </viewController>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="dkx-z0-nzr" sceneMemberID="firstResponder"/>
+            </objects>
+            <point key="canvasLocation" x="1069.2307692307693" y="3.5545023696682461"/>
+        </scene>
+        <!--Navigation Controller-->
+        <scene sceneID="uMq-uT-T57">
+            <objects>
+                <navigationController automaticallyAdjustsScrollViewInsets="NO" id="f9M-Ma-wfR" sceneMemberID="viewController">
+                    <toolbarItems/>
+                    <navigationBar key="navigationBar" contentMode="scaleToFill" id="sTW-Wh-vzE">
+                        <rect key="frame" x="0.0" y="47" width="390" height="44"/>
+                        <autoresizingMask key="autoresizingMask"/>
+                    </navigationBar>
+                    <nil name="viewControllers"/>
+                    <connections>
+                        <segue destination="BYZ-38-t0r" kind="relationship" relationship="rootViewController" id="Nwn-Pp-7gI"/>
+                    </connections>
+                </navigationController>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="xaQ-vv-7Bd" userLabel="First Responder" customClass="UIResponder" sceneMemberID="firstResponder"/>
+            </objects>
+            <point key="canvasLocation" x="140" y="3.5545023696682461"/>
+        </scene>
+    </scenes>
+    <resources>
+        <systemColor name="systemBackgroundColor">
+            <color white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+        </systemColor>
+        <systemColor name="systemOrangeColor">
+            <color red="1" green="0.58431372549019611" blue="0.0" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+        </systemColor>
+        <systemColor name="systemRedColor">
+            <color red="1" green="0.23137254901960785" blue="0.18823529411764706" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+        </systemColor>
+        <systemColor name="systemTealColor">
+            <color red="0.18823529411764706" green="0.69019607843137254" blue="0.7803921568627451" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+        </systemColor>
+    </resources>
+</document>
```

Comment:
May I suggest we use code for defining the UI instead of storyboards and use SwiftUI Previews to visualize. Less likely to have merge conflicts and ui code will be easier to update, reuse and reason about.

Reply:
We still support iOS 11.4 on those binaries. Not sure if we should use iOS 13 code samples on it.

Reply:
No worries, was just a suggestion.